Formula/libmobi.rb

```diff
@@ -0,0 +1,28 @@
+class Libmobi < Formula
+  desc "C library for handling Kindle (MOBI) formats of ebook documents"
+  homepage "https://github.com/bfabiszewski/libmobi/"
+  url "https://github.com/bfabiszewski/libmobi/releases/download/v0.6/libmobi-0.6.tar.gz"
+  sha256 "c35bd44279575bf8b102d23eba482805bfc1e5a49df8414d851507e8ea811c5d"
+  license "LGPL-3.0-only"
```

Comment:
```suggestion
  license "LGPL-3.0-or-later"
```
According to the code headers.